voluncheer/profiles/views/organizations.py

```diff
@@ -20,6 +19,15 @@ def get_context_data(self, **kwargs):
 
     def form_valid(self, form):
         """Saves the new user and logs them in."""
-        user = form.save()
-        login(self.request, user)
-        return redirect("profile")
+        user = form.save(commit=False)
+        user.is_active = True
+        user.save()
+        organization_profile = Organization.objects.create(
+            user=user,
+            name=form.cleaned_data.get("name"),
+        )
+        organization_profile.save()
+        user.is_active = False
```

Comment:
Same as below. I don't think you should touch is_active, unless it's a custom field.

Reply:
Same as above.